### PR TITLE
Polymer-2-hybrid-ify

### PR DESCRIPTION
**Pull request by ryantmer**
Bump to Polymer^1.8.0 and add various polymer-cli stuff. Attribute removal was caught by the linter; the attribute in question doesn't exist in the WC.

### CODE REVIEW

**Comment by ryantmer** [comment]
Not really a functional change, so shouldn't really need much review or anything - already verified that this doesn't change anything functionally-speaking, but a sanity check would be 👌

File: package.json [hunk 1]
```diff
@@ -3,18 +3,19 @@
   "description": "Polymer-based web component for a D2L image selector",
   "private": true,
   "scripts": {
-    "test": "npm run test:lint:js && npm run test:lint:wc && wct",
-    "test:lint:js": "eslint --ext .js,.html .",
-    "test:lint:wc": "polylint --no-recursion --input *.html",
+    "postinstall": "bower install",
+    "test": "npm run test:lint:js && npm run test:lint:wc && polymer test",
+    "test:lint:js": "eslint --ext .js,.html . test/",
```
Comment (cgalvind2l) [inline package.json]: Is there a reason the tests weren't being linted before?

Comment (ryantmer) [inline package.json]: I don't think so, I think they just tend to get missed 😢

**Comment by ryantmer** [comment]
Cross-browser is good idea. One set of cross-browser checks coming up!

**Comment by cgalvind2l** [comment]
Also, I notice there's no Travis on here doing checks - confirm that linting and unit tests don't fail?

**Comment by ryantmer** [comment]
Totally missed that. I'll add that too, I've added it in a few repos recently.

**Comment by ryantmer** [comment]
Hokay, cross-browser looks good (FF/Chrome/Edge/Safari/IE11). Clicking "Change Image" opened the change image overlay, the images loaded, and I could set a new image. Searching also worked as before.

I've got the .travis.yml ready, but this repo isn't enabled on Travis - need to find somebody that can do that first, then I'll push it.

**Pull request merged**